src/Microsoft.IdentityModel.JsonWebTokens/Json/JsonClaimSet.cs

```diff
@@ -422,6 +459,17 @@ internal bool TryGetClaim(string key, string issuer, out Claim claim)
         /// <returns></returns>
         internal bool TryGetValue<T>(string key, out T value)
         {
+#if NET7_0_OR_GREATER
```

Comment:
it is a little easier to understand if you ifdef these at the method level.

Reply:
Will refactor.